Makefile

```diff
@@ -13,7 +13,7 @@ export TERRAFORM_PROVIDER_DOWNLOAD_NAME:=terraform-provider-aviatrix
 export TERRAFORM_NATIVE_PROVIDER_BINARY:=terraform-provider-aviatrix_v3.1.0
 export TERRAFORM_DOCS_PATH:=docs
 
-PLATFORMS ?= linux_amd64 linux_arm64
+PLATFORMS ?= linux_amd64
```

Comment:
Any reason to not support multi-architecture? This would mean the provider may not run in some cloud providers depending on the instance type.

Reply:
I was not able to get the provider to successfully publish on arm64, do to missing dependencies in the up and related packages.  I may try to add that later, but for now I think it's best to leave this disabled.

Reply:
Gotcha - I'll leave it to your discretion if you want to note this in the readme or not while we fast-follow on this.